src/sentry/api/bases/sentryapps.py

```diff
@@ -321,3 +321,24 @@ def has_object_permission(self, request, view, installation):
 class SentryAppAuthorizationsBaseEndpoint(SentryAppInstallationBaseEndpoint):
     authentication_classes = (ClientIdSecretAuthentication, )
     permission_classes = (SentryAppAuthorizationsPermission, )
+
+
+class SentryInternalAppTokenPermission(SentryPermission):
+    scope_map = {
+        'POST': ('org:read', 'org:integrations', 'org:write', 'org:admin'),
```

Comment:
Why can someone with `read` scope make writes/deletes?

Reply:
@markstory We wanted to use the same permissions on the tokens that we use for creating a Sentry app. See the `SentryAppsPermission` class. That being said, maybe it makes sense to make a constant out of these four scopes and use it there as well so we have a single source of truth for these scopes. What do you think?

Reply:
@mnoble would like your opinion on creating a constant for the permissions as well

Reply:
I think we still limit `DELETE` https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/sentryapps.py#L126